src/plugins/intel_npu/src/compiler/include/zero_compiler_in_driver.hpp

```diff
@@ -132,14 +132,14 @@ class LevelZeroCompilerInDriver final : public ICompiler {
     void getNativeBinary(ze_graph_dditable_ext_curr_t& graphDdiTableExt,
                          ze_graph_handle_t graphHandle,
                          std::vector<uint8_t>& blob,
-                         uint8_t*& blobPtr,
+                         const uint8_t*& blobPtr,
```

Comment:
Interesting combination. 

If I read it correctly, this semantic implies that we can change the pointer itself, but want to keep the data, pointer is pointing to, constant. Is it really required for this case?

Reply:
I changed this because it was also changed in the driver. And pfnGetNativeBinary2 requires a const pointer.

```
typedef ze_result_t (ZE_APICALL *ze_pfnGraphGetNativeBinary_ext_2_t)(
    ze_graph_handle_t hGraph,                       ///< [in] handle of the graph object
    size_t* pSize,                                  ///< [out] size of native binary in bytes
    const uint8_t** pGraphNativeBinary              ///< [out] double pointer to view of native binary, driver owns the mem
```ory
    );